app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js

```diff
@@ -9,6 +10,21 @@ export default class extends Controller {
   @tracked selectedSidebarCategories = [];
   @tracked selectedSidebarTagNames = [];
 
+  constructor() {
+    super(...arguments);
+
+    this.sidebarListDestinations = [
+      {
+        name: I18n.t("user.experimental_sidebar.list_destination_default"),
+        value: "default",
+      },
+      {
+        name: I18n.t("user.experimental_sidebar.list_destination_unread_new"),
+        value: "unread_new",
+      },
+    ];
+  }
```

Comment:
Setting a default value for a property in a constructor is a little strange when we can just set the property directly when declaring the property.